client/docs/sections.test.js

```diff
@@ -0,0 +1,69 @@
+import { ensureParentSections, buildSectionTree, processDocumentFile } from "./sections";
```

Comment:
Thanks for adding a test; this and actually publishing it on updates in the future will keep this thing functional for longer than a month or two this time :)

Reply:
I don't think it actually runs where it is though, haha. I built it in a place where you can run jest, then just moved it there so we had it. We should probably figure out how to make it run where it is.

Reply:
Fixed. Jest now runs in docs too.